docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+- [ ] Identify the list of relevant Key Performance Indicators (KPIs) and metrics to identify what a successful launch would look like for each program launch. Example metrics:  
+      - [ ] Reduction in time to complete an application.  
+      - [ ] Increase in residents’ access to annual savings / benefits (dollars).  
+      - [ ] Increase in number of programs online / added to CiviForm.  
+- [ ] Identify key stakeholders throughout the community (eg. CBOs) and local governments who might be able to connect target user groups with the program to ensure a successful launch.  
+- [ ] If resident applications are managed or reviewed in an external case management system, work with your technical team to ensure necessary data pipelines are set up and tested.  
+- [ ] Develop a plan for thorough testing: Conduct comprehensive testing to identify and fix issues before launch.  
+- [ ] Train program administrators how to use CiviForm and how to route questions / issues they have.   
+- [ ] Perform user experience studies.  
+      - [ ] Ensure that at least 10 real end users have tested the new program.  
+      - [ ] Work with specific programs to test the end-to-end process for application submission and processing and make adjustments as needed.  
+- [ ] Pilot program: Plan a pilot launch to gather feedback and make improvements.  
+- [ ] Decide on launch announcement plans to ensure that residents can find and apply for the new streamlined programs.  
+      - [ ] Identify target audience/user group, communication channels (eg. press release, website announcement, social media posts, a newsletter) to share the news with residents.  
+      - [ ] Develop a communication plan: Inform citizens and stakeholders about the new system and its benefits and consider how you may drive awareness about the program.  
+      - [ ] Consider relevant metrics to measure if you are reaching your residents.  
+- [ ] Generate excitement by recording demos and sharing any initial findings, positive data outcomes, or testimonials resulting from use of CiviForm to connect users to the program.   
+- [ ] **Announce CiviForm to your community**   
+- [ ] Put ample opportunities for feedback in place during and after the launch so that users can flag any questions or bugs.   
+- [ ] If programs will be rolled out in phases, provide users with a timeline of future planned launches for additional programs.
+
+## Next steps \- Ongoing Evaluation and Improvement
+
+- [ ] Performance metrics: Track key performance indicators (KPIs) to measure the impact of CiviForm.  
+- [ ] User feedback: Gather feedback from citizens and staff to identify areas for improvement.  
+- [ ] Iterative development: Continuously update and refine forms based on user needs and data analysis.  
+- [ ] Continue to grow CiviForm, adding new programs over time.
+
+## Check out these links to read more about CiviForm
+
+* CiviForm home page (To Be Linked)  
```

Comment:
**nitpick:** It may be helpful to add "TODO" with a bug number (if it exists) to help you find these.